### PR TITLE
[plugin.video.regiotv@matrix] 0.1.3+matrix.1

### PR DESCRIPTION
**Pull request by dagwieers**
### Add-on details:

- **General**
  - Add-on name: Regio TV
  - Add-on ID: plugin.video.regiotv
  - Version number: 0.1.3+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/add-ons/plugin.video.regiotv
  
The Regio TV add-on lists all available Flemish regional television stations.

Available channels: ATV, BRUZZ, Focus TV, ROB-tv, TVL, TV Oost, WTV

[I]This add-on is not endorsed by any television station, and is provided 'as is' without any warranty of any kind.[/I]

### Description of changes:

v0.1.3 (2021-03-04)
- Fix an exception on Matrix when syncing channel/tvguide from IPTV Manager
- Remove WTV, combine with Focus TV

v0.1.2 (2020-08-01)
- Add support for IPTV Manager
- Add support for referer
- Add RegioTV artwork

v0.1.1 (2020-02-11)
- Fix regional channels BRUZZ, Focus TV and WTV

v0.1.0 (2019-11-08)
- Initial public release
  

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.regiotv/resources/lib/addon.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def main_menu():
         label = '{name} [COLOR gray]| {label}[/COLOR]'.format(**channel)
         plot = '[B]{name}[/B]\nRegio {label}\n\n[I]{description}[/I]\n\n[COLOR yellow]{website}[/COLOR]'.format(**channel)
         list_item = ListItem(label=label, label2=channel.get('label'), offscreen=True)
+        list_item.setProperty(key='IsInternetStream', value='true' if is_playable else 'false')
```
Comment (enen92) [inline plugin.video.regiotv/resources/lib/addon.py:33]: why this change? Afaik this is an internal kodi property. Can't you set the mimetype instead?

Comment (dagwieers) [reply plugin.video.regiotv/resources/lib/addon.py:33]: I cannot remember what I attempted to do here, but it makes no difference.
I have removed it upstream, not sure if you are blocking the release for this?

Comment (enen92) [reply plugin.video.regiotv/resources/lib/addon.py:33]: I'd prefer if you remove this yes. I have requested this in the past too, but it keeps being added as part of other bug fix updates...

Comment (dagwieers) [reply plugin.video.regiotv/resources/lib/addon.py:33]: Ok, I will make a new releease 🙄

Comment (dagwieers) [reply plugin.video.regiotv/resources/lib/addon.py:33]: No doubt it originated from the same add-on it was copied from. It has been part of this add-on since its inception.
